src/OSPSuite.Presentation/Services/ParameterIdentificationTask.cs

```diff
@@ -146,6 +160,9 @@ public void AddSimulationsTo(ParameterIdentification parameterIdentification, IE
 
       public void AddParameterTo(ParameterIdentification parameterIdentification, IParameter parameter)
       {
+         if (!_parameterSelector.CanUseParameter(parameter))
```

Comment:
The real only change. Parameter can only be added to PI if it can actually be used in a PI :)